### PR TITLE
fix: kind cluster config member2

### PR DESCRIPTION
**Pull request by shauvet**
**What type of PR is this?**

**What this PR does / why we need it**:
For local develepment
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: artifacts/kindClusterConfig/member2.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 kind: Cluster
 apiVersion: "kind.x-k8s.io/v1alpha4"
 networking:
+  apiServerAddress: "{{host_ipaddress}}"
```
Comment (RainbowMango) [inline artifacts/kindClusterConfig/member2.yaml:5]: @shauvet Can you explain why we need this?

Comment (shauvet) [reply artifacts/kindClusterConfig/member2.yaml:5]: First, when I did local development, without it, I couldn't start my local api server, second, I thought it should be similar with the member1's config. Do I understand wrong?

Comment (RainbowMango) [reply artifacts/kindClusterConfig/member2.yaml:5]: Thanks. @shauvet Could you please sign the commit as per https://github.com/karmada-io/dashboard/pull/87/checks?check_run_id=28865035951?

**Comment by warjiang** [comment]
@shauvet can you resolve the DCO problem ?

File: artifacts/kindClusterConfig/member2.yaml (hunk 1 above)
Comment (shauvet) [reply artifacts/kindClusterConfig/member2.yaml:5]: > Thanks. @shauvet Could you please sign the commit as per https://github.com/karmada-io/dashboard/pull/87/checks?check_run_id=28865035951?

OK, done.

**Comment by shauvet** [comment]
> @shauvet can you resolve the DCO problem ?

OK, it's done now.

**Pull request merged**